pyupgrade/_plugins/exceptions.py

```diff
@@ -0,0 +1,11 @@
+from __future__ import annotations
```

Comment:
I would probably just put all the exception rewriting here -- rather than making importable helpers for the other modules.  should simplify the structure a bit (and make this module less confusing since it's not actually a plugin)